### PR TITLE
Update 07-forest_plot.ipynb

### PR DESCRIPTION
**Pull request by JAnns98**
fixed some spelling mistakes

### CODE REVIEW

**Comment by Jacobluke-** [comment]
Found too many typos inside the notebook. Have to check again.

**Comment by JAnns98** [comment]
> Found too many typos inside the notebook. Have to check again.

Yeah I found more also

**Comment by Lucas1213WZY** [comment]
Sorry for making so many typo errors, I was juggling between my final project, dabest forest plot and midterms at the same time. Should I do an overall check today and change any mistakes or spelling errors found?

File: nbs/tutorials/07-forest_plot.ipynb [hunk 1]
```diff
@@ -17,9 +17,9 @@
    "id": "cfdb7e31",
```
Comment (Jacobluke-) [inline nbs/tutorials/07-forest_plot.ipynb:17]: <h3>Example for using reviewnb to modify the tutorial notebook.</h3><p> </p><p>The forest plot function is added later, so technically it's not supported neither with the updates of DABEST v2023.02.14 nor v2024.03.29, please consider a change or placeholder to remind the change of subsequent version number</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ACCLAB/DABEST-python/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (Lucas1213WZY) [reply nbs/tutorials/07-forest_plot.ipynb:17]: <p>I've changed it to a place holder like 20xx.xx.xx, is this scenario ok to include in the tutorial notebook?</p>

**Comment by Lucas1213WZY** [comment]
<p>Should we create a flow or a experiment that flows better? I was wondering how we would structure it.</p>

File: nbs/tutorials/07-forest_plot.ipynb (hunk 1 above)
Comment (Lucas1213WZY) [inline nbs/tutorials/07-forest_plot.ipynb:17]: <p>As beautiful as a dabest plot is, sometimes we want to alter its appearance, be it to change the position of the legend, use different texts in the legend or the axis labels, to add or remove lines etc, or to change the color and alpha of some particular graphical elements. Sometimes we even want to move parts of the plot, such as the bootstrapped half violins onto a different axes for a meta-analysis.</p><p> </p><p>Will this be a good introduction to add into the tutorial notebook as I copied this from PostHoc.ipynb? I will adapt these wordings maybe, to make it more smooth when introducing forest plot functionalities.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ACCLAB/DABEST-python/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by Jacobluke-** [comment]
@Lucas1213WZY I have merged the apiTut branch into this PR, you will start making commits on the tutorial notebook as per the reviews we have here.

**Comment by Lucas1213WZY** [comment]
> @Lucas1213WZY I have merged the apiTut branch into this PR, you will start making commits on the tutorial notebook as per the reviews we have here.

**Comment by Lucas1213WZY** [comment]
@Jacobluke- Thank you Zinana, I will work on the edits here

**Pull request merged**